modules/quickstart-uyuni/nav-quickstart-uyuni-guide.adoc

```diff
@@ -0,0 +1,26 @@
+// Quickstart Guide - Uyuni Antora Navigation List
+
+// PDF Installation Guide
+ifdef::backend-pdf[]
+= Quick Start - Uyuni: {productname} {productnumber}
```

Comment:
This probably needs checking.

Reply:
Checking for what, exactly? It'll only get built when we build Uyuni, so it will only use Uyuni version numbers.

Reply:
I think this will come out as:

> Quick Start - Uyuni:  Uyuni 2021-02

Maybe, product number will look a little bit different.  I'd rather go for a meaningful sub-title such as "Server Installation".  But it is a minor issue and we can adjust it when all Uyuni guides are available.

Reply:
Oh, I see what you're saying. Yes, you're probably right. We can take a closer look at this kind of thing when we do the guide enablement.